src/main/kotlin/com/demonwav/mcdev/platform/mixin/inspection/MixinClassReferenceInspection.kt

```diff
@@ -41,7 +41,7 @@ class MixinClassReferenceInspection : MixinInspection() {
                 return
             }
 
-            holder.registerProblem(type, "${referencedClass.name} is a Mixin class, and cannot be referenced directly")
+            holder.registerProblem(type, "Mixin class cannot be referenced directly")
```

Comment:
This should be really in #123, not sure why this is here. Anyway, the reason I did this is because the message seemed a bit long to me, in most cases you have the class name directly highlighted in the editor anyway (no need to display it again).